pkg/odo/cli/delete/component/component.go

```diff
@@ -331,16 +362,28 @@ func listResourcesMissingFromDevfilePresentOnCluster(componentName string, devfi
 }
 
 // printDevfileResources prints the devfile components for ComponentOptions.deleteDevfileComponent
-func printDevfileComponents(componentName, namespace string, k8sResources []unstructured.Unstructured) {
+func printDevfileComponents(
+	componentName, namespace string,
+	k8sResources []unstructured.Unstructured,
+	podmanResources []*corev1.Pod,
+) {
 	log.Infof("This will delete %q from the namespace %q.", componentName, namespace)
```

Comment:
Can you fix this message to also indicate podman if podmanResources are present?